src/Tree/Document/MongoDB/Repository/AbstractTreeRepository.php

```diff
@@ -72,25 +72,25 @@ public function __construct(DocumentManager $em, UnitOfWork $uow, ClassMetadata
     }
 
     /**
-     * Sets the RepositoryUtilsInterface instance
```

Comment:
Why do we need this change?

Reply:
Ok! It has been suggested by PHPCS after I rebased main, it was one of the breaking points during CI checks.

Reply:
Cleanup done